### ISSUE TITLE
Clarification around enabling complete checking and Swift 6

### ISSUE THREAD

**Issue by mattmassicotte** [apple/swift-migration-guide#19]
I think `CompleteChecking.md` may need a little more details around to enable complete checking with a Swift 6.

If I follow the guide as written now, I get an error with the Swift 6 toolchain with6 mode explicitly enabled in the manifest:

```
Building for debugging...
error: emit-module command failed with exit code 1 (use -v to see invocation)
<unknown>:0: error: upcoming feature 'StrictConcurrency' is already enabled as of Swift version 6
```

**Comment by mattmassicotte** [comment]
I think this only comes up for v6 targets. As long as you are still compiling with the 5 language mode, this is the correct thing to do.

**Comment by mattmassicotte** [comment]
Fixed with #40
